lib/Alembic/AlembicConfig.cmake.in

```diff
@@ -2,7 +2,12 @@
 
 include(CMakeFindDependencyMacro)
 # TODO whenever we loose the back-compatibility with IlmBase < 3, a REQUIRED needs to be added to find_dependency()
-find_dependency(Imath)
+if (@ALEMBIC_USING_IMATH_3@)
+    find_dependency(Imath)
+else ()
+    # Compatibility with OpenEXR 2.x, prior to Imath 3.x
+    find_dependency(IlmBase)
```

Comment:
Put it like that I think you can now add the `REQUIRED` to `find_dependency`.
I think it is the right way to deal with two possible versions. Until, if ever, Imath3 will diverge and becomes incompatible with the previous version (API changes and so on). When/If it happens it will be probably better to drop the support for 2.x and focus only on the recent version, it will make things much easier (IMO).

```suggestion
    find_dependency(Imath REQUIRED)
else ()
    # Compatibility with OpenEXR 2.x, prior to Imath 3.x
    find_dependency(IlmBase REQUIRED)
```

Reply:
My hunch on making it REQUIRED is it could cause problems with certain OpenEXR 2 setups that doest provide the CMake breadcrumbs.

Reply:
Currently (prior to this patch), everything is fine for a downstream package when using Alembic built against Imath3, but is broken when consuming Alembic's exported cmake config if that Alembic (and the downstream app) are built against OpenEXR 2.x (per VFX Platform 2020 & 2021).

With this patch, the case that worked before (building against Imath3) is identical. The previously-broken case can't be any worse, and as far as I can tell, it is fixed.

Proposal: Let's leave this patch as-is for right now, it fixes one case and seems unlikely to break anything that wasn't already broken. Afterwards, you can return to the "REQUIRED" question separately, with more attention to whether that might cause new problems for downstream projects.

Reply:
Yup, sounds good, I can introduce REQUIRED in a future PR that also includes an install test.